modules/sdpolicy-security.adoc

```diff
@@ -58,6 +57,4 @@ See link:https://www.openshift.com/products/dedicated/process-and-security#compl
 
 [id="network-security_{context}"]
 == Network security
-With {product-title} on AWS, AWS provides a standard DDoS protection on all Load Balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing Load Balancers used for {product-title}. We also add a 10 second timeout for HTTP requests coming to our haproxy router to receive a response or the connection is closed to provide additional protection.
-
-// TODO: Rewrite to avoid "we/our"
+With {product-title} on AWS, AWS provides a standard DDoS protection on all Load Balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing Load Balancers used for {product-title}. A 10 second timeout is added for HTTP requests coming to the haproxy router to receive a response or the connection is closed to provide additional protection.
```

Comment:
s/10 second timeout/10-second timeout (use a hyphen for clarity in a compound adjective)

Reply:
Resolved